### ISSUE TITLE
Export functionality from "deprecated" imported file types not functioning correctly.

### ISSUE THREAD

**Issue by OluAgunloye** [mockoon/mockoon#659]
**Describe the bug**
On Windows a related issue to #647  persists, any import of a now deprecated format seems to import correctly, but when you want to export to the "new" (formats supported as of 1.18.0) format, the export "loses" data. 

**To Reproduce**
1. Import file of deprecated type (in either OpenAPIv3 or Mockoon format). Be sure to use rules in this file type. This works correctly. 
2. Export the loaded environment in OpenAPIv3/Swaggerv2 format. This is done incorrectly, apparently.
3. Attempt to load (re-import) the previously newly exported environment and observe that rule sets as well as the response bodies have been removed. The file representing the re-imported environment is significantly smaller (in size) than the previously imported version (from step 1).

**Expected behavior**
That is not expected, as what Mockoon imports correctly it should surely be able to export correctly. 

**Screenshots**
None needed, you can use the sample routes Mockoon provided prior to v1.18.0 to observe this. 

**Mockoon version:**
1.18.0

**OS / OS version:**
Windows 10

**Comment by 255kb** [comment]
I think there is a misunderstanding here. 
Let me clarify things: 
- Mockoon stores each environment in a JSON file (right click on an environment name to find the file on your hard drive). 
- Before v1.18.0, there was an import/export option **in Mockoon's format** which was removed. It was basically a JSON format wrapping the environment. Nothing fancy, and a bit useless since now Mockoon is treating each environment separately. 
- From the begining we had a import/export in OpenAPI format which is NOT related to the previous one. The OpenAPI specification is not completely supported/implemented in Mockoon for multiple reasons: 
  - Mockoon is NOT an API design application, so, some OpenAPI features have NO equivalent in Mockoon
  - Some Mockoon's features have NO equivalent in OpenAPI: I think there is no concept of rules in OpenAPI at least not how they are working in Mockoon
  - Some things are lost when exported to the OpenAPI format, like the body, as there may be templating (`{{xxxx}}`) which may make it hard to create an object schema. 

TL;DR
- import/export in OpenAPI format is NOT related to the old import/export that was removed
- there is not a 1:1 match between the OpenAPI specs and Mockoon capabilities

I hope this clarifies a bit how Mockoon works. 

You can find an issue [here](https://github.com/mockoon/mockoon/issues/352) tracking the missing body example when exporting to the OpenAPI format.
